org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current ambient temperature -->
+    <method name="getRoomTemperature">
```

Comment:
I actually liked "getCurrentTemperature" (what you had before) better. I think "getRoomTemperature" could be a bit confusing in some cases. For example my thermostat (an ecobee) can report separate temperature readings for multiple rooms and average (a subset of) them together to get the "current" temperature.